contracts/TokenVault.sol

```diff
@@ -178,6 +189,36 @@ contract TokenVault is Ownable, Recoverable {
     return token.balanceOf(address(this));
   }
 
+  /// @dev Check how many tokens "investor" can claim
+  /// @param investor Address of the investor
+  /// @return uint How many tokens the investor can claim now
+  function getCurrentlyClaimableAmount(address investor) public constant returns (uint claimableAmount) {
+    uint maxTokensLeft = balances[investor] - claimed[investor];
```

Comment:
I would rename this `maxTokensLeft` to something like `maxClaimableAmount` or `totalClaimableAmount` or even `totalClaimableAmountLeft`. Main reason is `tokensLeft` is already descriptive enough for this context but not align with the wording `amount` here and there is nothing to compare `max` with